Microsoft.Extensions.ApiDescription.Client.targets on Microsoft.Extensions.ApiDescription.Client NuGet package has invalid XML

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When trying to build for multi-targeting, `src/Tools/Extensions.ApiDescription.Client/src/buildMultiTargeting/Microsoft.Extensions.ApiDescription.Client.targets` is used and contains an error at [line  18]:

```xml
      Condition=" ''$(OpenApiGenerateCodeOnBuild)' == 'true' "
```

that causes:

```
`...\.nuget\packages\microsoft.extensions.apidescription.client\6.0.24\buildMultiTargeting\Microsoft.Extensions.ApiDescription.Client.targets(18,7): error MSB4092: An unexpected token "$(OpenApiGenerateCodeOnBuild)" was found at character position 4 in condition " ''$(OpenApiGenerateCodeOnBuild)' == 'true' ".`
```


### Expected Behavior

No syntax errors in published artifacts.

### Steps To Reproduce

Create a project multi-targeting more than one TFM and reference package `Microsoft.Extensions.ApiDescription.Client`.

### Exceptions (if any)

```
`...\.nuget\packages\microsoft.extensions.apidescription.client\6.0.24\buildMultiTargeting\Microsoft.Extensions.ApiDescription.Client.targets(18,7): error MSB4092: An unexpected token "$(OpenApiGenerateCodeOnBuild)" was found at character position 4 in condition " ''$(OpenApiGenerateCodeOnBuild)' == 'true' ".`
```

### .NET Version

Any version of the package I've looked at.

### Anything else?

_No response_

--- a/src/Tools/Extensions.ApiDescription.Client/src/buildMultiTargeting/Microsoft.Extensions.ApiDescription.Client.targets
+++ b/src/Tools/Extensions.ApiDescription.Client/src/buildMultiTargeting/Microsoft.Extensions.ApiDescription.Client.targets
@@ -15,6 +15,6 @@
 
   <Target Name="_GenerateOpenApiCode"
       BeforeTargets="BeforeCompile"
-      Condition=" ''$(OpenApiGenerateCodeOnBuild)' == 'true' "
+      Condition=" '$(OpenApiGenerateCodeOnBuild)' == 'true' "
       DependsOnTargets="GenerateOpenApiCode" />
 </Project>

